python/lsst/sims/GalSimInterface/galSimCelestialObject.py

```diff
@@ -61,6 +67,9 @@ def __init__(self, galSimType, sed, ra, dec, xPupil, yPupil,
         self._majorAxisRadians = majorAxis
         self._positionAngleRadians = positionAngle
         self._sindex = sindex
+        self._g1 = gamma1/(1. - kappa)
```

Comment:
Could you add a comment explaining what g1, g2, and mu are?  It might also be good to include a reference to their relationship with gamma1, gamma2, and kappa.

Reply:
Ok, will do.